GNUmakefile

```diff
@@ -15,6 +15,9 @@ tools:
 build: fmtcheck vet
 	go install
 
+build-dbg:
```

Comment:
why is this required in MAKEfile?

Reply:
you need to disable compiler optimizations to produce debuggable binary, so you need to pass thos `gcflags`. It is easier to remember `make build-dbg` vs `go install -gcflags "all=-N -l"`